roles/install_dbserver/tasks/EPAS_Debian_install.yml

```diff
@@ -0,0 +1,45 @@
+---
+- name: Install EPAS Packages
+  package:
+    name:
+      - python3-pip
+      - python3-psycopg2
```

Comment:
Are those packages available on Ubuntu18 and Debian9? I'm asking because we install other packages in the next task.

Reply:
Yes. As per the testing, all the packages are available for Ubutu18 and Debian 9